### ISSUE TITLE
Tests are currently failing

### ISSUE THREAD

**Issue by ModeSevenIndustrialSolutions** [os-climate/ITR-examples#133]
Please see:
[https://github.com/os-climate/ITR-examples/actions/runs/8454294187/job/23158978302?pr=131](https://github.com/os-climate/ITR-examples/actions/runs/8454294187/job/23158978302?pr=131)

Looks like the coverage reporting and/or test setup may need checking.

```
============================= test session starts ==============================
platform linux -- Python 3.9.19, pytest-7.3.2, pluggy-1.4.0
rootdir: /home/runner/work/ITR-examples/ITR-examples
configfile: pyproject.toml
plugins: mock-3.14.0, anyio-4.3.0, cov-5.0.0, dash-2.14.1
collected 4 items

/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/coverage/inorout.py:503: CoverageWarning:

Module ITR-examples was never imported. (module-not-imported)

/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/coverage/control.py:887: CoverageWarning:

No data was collected. (no-data-collected)

/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pytest_cov/plugin.py:339: CovReportWarning:

Failed to generate report: No data to report.


test/test_target_plots.py ....                                           [100%]WARNING: Failed to generate report: No data to report.



=============================== warnings summary ===============================
test/test_target_plots.py::TestTargets::test_different_starting_dates_intensity
test/test_target_plots.py::TestTargets::test_target_[20](https://github.com/os-climate/ITR-examples/actions/runs/8454294187/job/23158978302?pr=131#step:6:21)30
test/test_target_plots.py::TestTargets::test_target_netzero
  /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pandas/core/dtypes/cast.py:1565: UnitStrippedWarning:
  
  The unit of the quantity is stripped when downcasting to ndarray.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

---------- coverage: platform linux, python 3.9.19-final-0 -----------

FAIL Required test coverage of 95% not reached. Total coverage: 0.00%
=================== 4 passed, 3 warnings in 60.76s (0:01:00) ===================
Error: Process completed with exit code 1.
```

**Comment by ModeSevenIndustrialSolutions** [comment]
Also, current, Jupyter notebook builds.

**Comment by MichaelTiemannOSC** [comment]
ITR is a module that is properly imported and for which coverage is meaningful.  ITR-examples is just a collection of things in a repo that are independently meaningful, but not meaningful as a module.  Coverage should be defined on a per-executable basis (such as ITR_UI.py) not on a per-module basis (ITR-examples is meaningless).

I don't know enough about how code coverage thinks about such things to begin to formulate a proper answer beyond the above observation.

**Comment by ModeSevenIndustrialSolutions** [comment]
We can probably just tweak the thresholds, then?

**Comment by ModeSevenIndustrialSolutions** [comment]
Fixed iwth PR #188
